ui/app/router.js

```diff
@@ -172,7 +168,10 @@ Router.map(function () {
           this.mount('ldap');
           this.mount('pki');
           this.route('index', { path: '/' });
-          this.route('configuration');
+          this.route('configuration', function () {
+            // only CONFIGURABLE_SECRET_ENGINES can be configured and access the edit route
+            this.route('edit');
```

Comment:
I double checked locally but it looks like we hadn't been using the `section` route under `configure-secret-backend` ✅

Reply:
That one threw me. I searched and searched, and came to the same conclusion.